search-app/docs/api.md

```diff
@@ -1145,6 +1145,14 @@ Find collections that match all of the 'project' param values
 
      curl "%CMR-ENDPOINT%/collections?updated_since=2014-05-08T20:06:38.331Z"
 
+#### <a name="c-created-at"></a> Find collections by created_at
```

Comment:
Add `created-at` to the `Collection Search By Parameters` section of the api doc.